<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointEdit</class>
 <widget class="QWidget" name="PointEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>90</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>255</width>
    <height>90</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>255</width>
    <height>90</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Редактирование точки</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_Parcel">
     <property name="text">
      <string>Посылка:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_Description">
     <property name="text">
      <string>Описание:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton_Clean">
     <property name="text">
      <string>Отчистить</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_Yes">
     <property name="text">
      <string>Да</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButton_No">
     <property name="text">
      <string>Нет</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_Description"/>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QSpinBox" name="spinBox_Parcel">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButton_No</tabstop>
  <tabstop>lineEdit_Description</tabstop>
  <tabstop>pushButton_Yes</tabstop>
  <tabstop>pushButton_Clean</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Clean</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_Description</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_No</sender>
   <signal>clicked()</signal>
   <receiver>PointEdit</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_Description</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton_Yes</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
